### ISSUE TITLE
Other Channels

### ISSUE THREAD

**Issue by dahartigan** [cabernetwork/provider_video_daddylive#1]
Hi, thanks for your work with Cabernet and this daddylive plugin!  Would it be possible to add the extra channels that are present in "Other Channels.txt" (https://github.com/cabernetwork/provider_video_daddylive/blob/main/provider_video_daddylive/resources/Other%20Channels.txt) into the main list also?

I'm aware of a hack, but updates break that and it would be awesome to have them all available in all versions on this repo.

Thanks again!

**Comment by dahartigan** [comment]
How often do the channels change? The main issue that I experience is that updates to Cabernet would break all of the extra channels every time Cabernet receives an update.  

I am proposing that you add the extra code from the hack, including the list.index from that hack, into this repo so that those channels would never break by an update again.

As for channels that change, if it's infrequent enough that an issue on github would suffice then I think this would still have a better experience for the user and yourself.  If it turns out that the channels change frequently, that would not be ideal and perhaps we would brainstorm an alternative.

Thanks for your time!

**Comment by rocky4546** [comment]
Local channels change at least monthly if not weekly. A few of the general channels change time zones around monthly. The provider adds new channels to the other list around monthly, so having updates monthly would be expected.

**Comment by rocky4546** [comment]
https://insolencetvgo.godaddysites.com/ is another site containing some of the channels listed in Other Channels.

**Comment by mdrichardson** [comment]
@rocky4546, I could probably help with this and write a script to help maintain this. However, with a brief glance at the code, I'm not sure I understand how `channel_list.json` works. What are the zones and ids? I think, ideally, these are grouped separately--daddylive doesn't list most of these on their "24/7" channels, but they are listed in the TV schedule API. Might be nice to have a separate toggle for enabling "non-24/7" channels. Any chance you can provide some pointers or very general steps for how to start on this?

**Comment by rocky4546** [comment]
Most tvguide providers talk about "providers".  For TVGuide, the providers are found here
https://www.tvguide.com/listings/
and then click on Provider.  Each provider has its unique zone id.  Also, each channel will have its own id (although a west coast channel would be different than an east coast channel since the guide would change.)  Zap2it follows the same format providing a id for the provider and channel.
Here are some examples... For a generic East Coast channel list, you can use
https://fandom-prod.apigee.net/v1/xapi/tvschedules/tvguide/serviceprovider/9100001138/sources/web
then here is a schedule you can get
https://fandom-prod.apigee.net/v1/xapi/tvschedules/tvguide/9100001138/web?start=1707427800&duration=180

**Comment by mdrichardson** [comment]
Figured I'd post an update...I started working on this, but ran into a couple of issues that have demotivated me from continuing (for now, at least):

1. There's no way to programmatically get all DL channel listings (via scrape or API). AFAIK, the only real complete list is in their Discord. The 24/7 list is missing some and their scheduling API only shows channels from that day, so it would need to be polled multiple times.
2. It really wouldn't make sense to update the channels frequently without pinning the channel numbers in Cabernet so that they match DL's. I'm sure it's possible to adjust the code to do this, but it looked like more substantial of a refactor than I have the time to contribute
